.circleci/config.yml

```diff
@@ -31,11 +31,17 @@ commands:
       - when:
           condition: << parameters.browsers >>
           steps:
-            - run:
-                name: Install pnpm package manager
-                command: |
-                  corepack enable
-                  corepack prepare pnpm@latest-8 --activate
+            - run: corepack enable
+      - when:
+          condition:
+            not: << parameters.browsers >>
```

Comment:
Why is this condition used?

Reply:
Because we get EACCESS errors on circleci with the `corepack enable` command: https://stackoverflow.com/a/73411601

And we `--install-directory ~/bin` breaks on the playwright docker image